traffic_ops/traffic_ops_golang/auth/authorize.go

```diff
@@ -39,6 +43,13 @@ type CurrentUser struct {
 	Capabilities pq.StringArray `json:"capabilities" db:"capabilities"`
 }
 
+type PasswordForm struct {
+	Username string `json:"u"`
+	Password string `json:"p"`
+}
+
+const disallowed = "disallowed"
```

Comment:
Shouldn't this be an Enum of the "roles" somewhere?

Reply:
no, unless we made the role names fixed, this is an assumption made on line 125, and noted in a comment, that should be fixed to use the DB in some way most likely.